src/test/regress/expected/multi_partitioning.out

```diff
@@ -1958,12 +1958,33 @@ SELECT parent_table, partition_column, partition, from_value, to_value FROM time
  public.non_distributed_partitioned_table | a                | public.non_distributed_partitioned_table_1 | 0          | 10
 (6 rows)
 
+-- create the same partition to verify it behaves like in plain PG
+CREATE TABLE partitioning_test_2011 PARTITION OF partitioning_test FOR VALUES FROM ('2011-01-01') TO ('2012-01-01');
+ERROR:  relation "partitioning_test_2011" already exists
+CREATE TABLE IF NOT EXISTS partitioning_test_2011 PARTITION OF partitioning_test FOR VALUES FROM ('2011-01-01') TO ('2012-01-01');
+NOTICE:  relation "partitioning_test_2011" already exists, skipping
+-- verify we can create a partition that doesn't already exist with IF NOT EXISTS
+CREATE TABLE IF NOT EXISTS partitioning_test_2012 PARTITION OF partitioning_test FOR VALUES FROM ('2012-01-01') TO ('2013-01-01');
+SELECT COUNT(*) FROM pg_dist_partition WHERE logicalrelid = 'partitioning_test_2012'::regclass;
+ count
+-------
+     1
+(1 row)
```

Comment:
Instead, we can verify this by checking the worker. You can connect to worker node and list shards, in test sql file by using `\c - - - :worker_1_port`. See `alter_index.sql` for an example.

Reply:
I am familiar with testing in the worker nodes from previous issues, but in this particular case I think using `pg_dist_partition` is more consistent here, considering the rest of `multi_partitioning.sql`.
For more consistence, I will modify my use of `pg_dist_partition` and make it the same as the other uses in the same file.